metadata/db.go

```diff
@@ -27,12 +27,17 @@ const (
 	dbVersion = 1
 )
 
+// DB represents a metadata database backed by a bolt
+// database. The namespace is fully namespaced and proxies
+// access to backend datastores for content and snapshots.
```

Comment:
Maybe mention this handles containers and images, as well as proxying to the backend for content and snapshots.